File: Recommender/src/main/java/cn/edu/ahtcm/caseclass/bean/Rating.scala
package cn.edu.ahtcm.caseclass.bean

/**
 * Rating 数据集
 *
 * @param userId    用户 ID
 * @param productId 商品 ID
 * @param score     评分
 * @param timestamp 时间戳
 */
case class Rating(userId: Int, productId: Int, score: Double, timestamp: Int)

File: Mall/src/main/java/cn/edu/ahtcm/mall/bean/Rating.java
package cn.edu.ahtcm.mall.bean;

import java.util.Date;

public class Rating {

    private int userId;
    private int productId;
    private double score;
    private long timestamp;

    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }

    public int getProductId() {
        return productId;
    }

    public void setProductId(int productId) {
        this.productId = productId;
    }

    public double getScore() {
        return score;
    }

    public void setScore(double score) {
        this.score = score;
    }

    public long getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(long timestamp) {
        this.timestamp = timestamp;
    }

}
